app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -25,6 +27,59 @@ def index
         end
       end
 
+      # Returns all pages as a tree from the root given by the id parameter
+      #
+      def tree
```

Comment:
I think we should move this into the `api` namespace. What do you think @mamhoff?